bokehjs/src/coffee/models/widgets/cell_editors.coffee

```diff
@@ -5,26 +5,24 @@ $2 = require "jquery-ui/spinner"
 
 p = require "../../core/properties"
 
+BokehView = require "../../core/bokeh_view"
 Model = require "../../model"
 
-Widget = require "./widget"
-
-class CellEditor extends Model
-
-class CellEditorView extends Widget.View
+class CellEditorView extends BokehView
 
   tagName: "div"
-  className: "bk-cell-editor"
+  attributes:
+    class: "bk-cell-editor"
```

Comment:
How this works? I don't see any references to `attributes` on non-Backbone.Model.